tdms/src/fields/td_field_exporter_2d.cpp

```diff
@@ -1,8 +1,10 @@
 #include "field.h"
 
 #include <stdexcept>
+#include <string>
```

Comment:
```suggestion
#include <string>

#include <spdlog/spdlog.h>
```
Explicit include of SPDLOG (it's not included in `field.h`)?